Toolchain setup description in contributing guidelines is incorrect if there is a pre-existing nightly installed

### General Summary
If there is a pre-existing `nightly` version of the rust toolchain installed, the [current contributing guidelines](https://github.com/luna/ide/blob/e4a9d7742a4e4d4d0620b5ffe1fd60261438adb2/docs/contributing.md) may fail due to the wrong `nightly` version being used to build the project.

### Steps to Reproduce
```
rustup toolchain install nightly  # This could have happened any time before setting up the IDE
rustup toolchain install nightly-2019-11-04
rustup default nightly            
node ./run build
```
Because my default  `nightly` is newer than `nightly-2019-11-04`.

### Expected Result
The build should succeed. The instructions should be robust and work in as many environments as possible, especially since the initial build can take a long time.

### Actual Result

```
error[E0557]: feature has been removed
  --> lib\shapely\impl\src\lib.rs:14:12
   |
14 | #![feature(overlapping_marker_traits)]
   |            ^^^^^^^^^^^^^^^^^^^^^^^^^ feature has been removed
   |
   = note: removed in favor of `#![feature(marker_trait_attr)]`
[...]
```

The instructions fail to build, as the latest pre-existing `nightly` version is used instead of the explicitly installed  `nightly-2019-11-04`  one. 

### Enso Version
master e4a9d7742a4e4d4d0620b5ffe1fd60261438adb2

--- a/docs/contributing.md
+++ b/docs/contributing.md
@@ -12,7 +12,7 @@ need the following setup:
 
   ```bash
   rustup toolchain install nightly-2019-11-04 # Install the nightly channel.
-  rustup default nightly                      # Set it as the default one.
+  rustup override set nightly-2019-11-04      # Set it as the default toolchain for this folder.
   rustup component add clippy                 # Install the linter.
   cargo install cargo-watch                   # To enable ./run watch utility
   ```

